src/Microsoft.CodeAnalysis.FxCopAnalyzers/Microsoft.CodeAnalysis.FxCopAnalyzers.md

```diff
@@ -2688,7 +2688,7 @@ By default, the Trusted Root Certification Authorities certificate store is conf
 |CodeFix|False|
 ---
 
-## [CA5382](https://docs.microsoft.com/visualstudio/code-quality/ca5382): Use Secure Cookies In ASP.Net Core
+## [CA5382](https://docs.microsoft.com/visualstudio/code-quality/ca5382): Use Secure Cookies In ASP.NET Core
```

Comment:
This is auto-generated file, please revert changes. These are actually resource strings provided by the analyzer.

Reply:
@Youssef1313 If you need to make a change here because something is wrong, you can do so by updating the original source that produces this file.